### PR TITLE
Tools - Update `make.py` for pboProject 2.63

### LINKED ISSUE #7350: make.py: -x flag not supported in pboProject 2.63
**Mods (complete and add to the following information):**
No mods

**Description:**

When running make.py gives the error "pboproject return code == 1"
http://prntscr.com/qidl3d
and does nothing

> При запуске make.py выдает ошибку "pboproject return code ==1" 
> http://prntscr.com/qidl3d
> и ничего не делает

**Steps to reproduce:**
install latest mikerotools version pboProject is 2.63.7.45 from
https://mikero.bytex.digital/api/download?filename=pboProject.2.63.7.45.Installer.exe
from https://mikero.bytex.digital/Downloads
install python
make drive P
run setup.py
run make.py

**Expected behavior:**

**Where did the issue occur?**

**Log Files:**

**Additional context:**
a problem with the -X flag, which, apparently, is not supported in newer versions of utilities
https://prnt.sc/qidoa9
There are no problems without this flag
http://prntscr.com/qidp2c
from pboProject.DosMode.htm documentation:
+ | -X no such thing you CANNOT affect exlusion of pbo files only uses list defined in gui -X = no such thing you CANNOT affect exlusion of pbo files

> проблема с флагом -х, который, видимо, не поддерживается в новых версиях утилит
> https://prnt.sc/qidoa9
> Без этого флага проблем нету
> http://prntscr.com/qidp2c
> из документации pboProject.DosMode.htm:
> +|-X   no such thing you CANNOT affect exlusion of pbo files   only uses list defined in gui    -X=   no such thing you CANNOT affect exlusion of pbo files     

**Screenshots:**
image
image
image

### PR DESCRIPTION
**Pull request by lexlaw2**
**When merged this pull request will:**
- Remove -X flag whitch not supporting with current version of Micero pboProject
- Fix #7350

### CODE REVIEW

**Comment by jonpas** [comment]
This just removes whatever that flag does.

Flag got renamed to `G`. And that's not backwards compatible, which sucks.

**Comment by jonpas** [comment]
What does that flag (old `-X` and `+X`, new `-G` and `+G`) do anyways?

**Comment by PabstMirror** [comment]
2.45
+|-X	deprecated external references	 
+|-G	external references	same as makepbo

2.63
+|-X	no such thing you CANNOT affect exlusion of pbo files
+|-G	enable/disable file reference checks

**Comment by jonpas** [comment]
So `X` never worked (deprecated, removed) and `G` are file reference checks that we don't want anyways?

File: tools/make.py [hunk 1]
```diff
@@ -1278,9 +1278,9 @@ def main(argv):
 
                     else:
                         if check_external:
-                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "+X", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
+                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
```
Comment (PabstMirror) [inline tools/make.py]:
```suggestion
                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "+G", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
```

File: tools/make.py [hunk 2]
```diff
@@ -1278,9 +1278,9 @@ def main(argv):
...
                         else:
-                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "-X", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
+                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
```
Comment (PabstMirror) [inline tools/make.py]:
```suggestion
                            cmd = [pboproject, "-P", os.path.join(work_drive, prefix, module), "+Engine=Arma3", "-S","+Noisy", "-G", "+Clean", "+Mod="+os.path.join(module_root, release_dir, project), "-Key"]
```

**Comment by PabstMirror** [comment]
X might have still worked because it wasn't checking ext files
right now it fails to build rhs compats because I don't have rhs on p:
also ace_hot uses a file from a encrypted pbo

**Comment by jonpas** [comment]
`G` should be linked to `check_external` argument of `make.py` then.

**Pull request merged**